### DISCUSSION TITLE
Remake Stealers/Troublemakers

### DISCUSSION THREAD

**Discussion by nasko222** [nasko222/the-minecraft-recreation-project#161] [Show and tell]
Ok I know this is a bit out of place, but I thought there should be a place to report remake thieves and others just so the TMRP community/enjoyers know about it. In this topic you can provide info of someone you believe is a remake thief or a troublemaker who false credits TMRP but they're showing very poor quality item. You can use the hash table to check whether a file is matching TMRP using the following command: "certutil -hashfile (filename.jar) MD5". Of course these can be manipulated and then you can check for source code matches. Of course most of the people who do that are kids and don't know better so they won't even bother obfuscating the hashmap. Also this topic will hopefully be helpful for other people who look at these videos and believe the versions are real.

And please don't go about Tristan Corso on this topic, if you don't know he is MEGA unhinged and whatever he talks just makes zero sense, Thats due to low functioning autism, and I really can't do a lot to stop him from stalking me or method or anyone from TMRP in this regard. xd (Heck just look at his twitter 💀 )

IMPORTANT NOTE: This is not witchhunting. There are actual people who make their own remakes and admire their own work, eventhough they're less than the troublemakers, Keep that in mind.

**Comment by nasko222** [comment]
Here's one I found today.

https://www.youtube.com/watch?v=79ivdJJxhUw&ab_channel=Minecraftmods%D0%B8%D1%8D%D0%BC%D0%BE%D0%B4%D0%B6%D0%B8%3A%29

I ran the 0.24 in certutil and it gave me exact hashmap. So that one is pretty obvious

![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/52720373/27b1a724-8de8-4197-b859-590232b1ca6b)

![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/52720373/c5f28113-3844-4d01-86c4-630515b76a8e)

Also the dude has a discord server where he is named "EMOJI222" That could be a coincidence but i found it disturbing, The image is also from MD3 test remake because that frame doesn't exist in the MD3 test video. It's clearly he is a TMRP enjoyer for the very least

![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/52720373/5fd34ccd-7249-4f91-935b-d09568139458)

**Comment by G23M08** [comment]
a youtube channel called "minecraft anomalies" made some videos with your recreations i think youve already seen it tho

**Comment by nasko222** [comment]
yep ive seen it
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/52720373/7add93bd-641f-4661-9446-1fb389d1c777)

**Reply by G23M08** [reply]
k

**Comment by G23M08** [comment]
https://www.youtube.com/watch?v=koSZw5pgc6w also did you let phobtdya edit your code?

**Comment by nasko222** [comment]
i know this dude, he asked me for remakes with PLS PLS GIVE ME PLEASE when they were private, this vid is 2 year ago so its not mine, plus 19a-dev is super easy to make

**Reply by G23M08** [reply]
yea

**Reply by G23M08** [reply]
also, i think I found an easy way to find these people go on YouTube and search the name of  the version and then add Minecraft before or after it

**Reply by nasko222** [reply]
just search minecraft remake or minecraft recreation then include classic or indev or infdev and then sort by upload date, also works. i do that every once in a while

**Reply by G23M08** [reply]
gex gaming is kida weird he sometimes credist you sometimes he dosent

**Reply by nasko222** [reply]
he is our friend and also contributor to the repo, we believe he has autism, but unlike tristan he actually cares about friends

**Reply by G23M08** [reply]
**k**

**Comment by nasko222** [comment]
easy as changing texture and version label

ALSO HAHAHAHAHAHAHA I JUST SAW. the sponge works. it shouldnt work in the dev builds, yeah thats a little bit hard to code (and it actually requires coding) XDXDXDXDXD

**Comment by G23M08** [comment]
https://www.youtube.com/watch?v=2hCSfAvV_EI

**Reply by nasko222** [reply]
lol wtf is this :DDDDDDDDDDDDDDD these are not my remakes btw

**Reply by G23M08** [reply]
what are the differences
how do i find out what is your remake by some videos

**Reply by nasko222** [reply]
you dont, i can know from experience, i can tell which remake is mine by just looking at it, because simply i coded it and i know what to expect. if something doesnt feel right i call it off. also you dont have to worry about troublemakers, me and my friends on the recreation station already covered that problem, dont look at old videos, if you want to, say something if you see a new video, like a brandly newly uploaded.

**Reply by G23M08** [reply]
k

**Reply by nasko222** [reply]
i envisoned this topic as me or someone from the contributors finding stuff like that, and the rest on github to be like "oh hell no wtf is this dude serious". dont feel pressured to watch 24/7. thats currently under control, its not like years ago we're prepared with tools and shit

also btw do you go by any other names (on discord for example), asking cuz im curious if youre familiar

**Reply by G23M08** [reply]
I used to watch methods videos but I do have a reddit with the same name

**Reply by nasko222** [reply]
nice

**Reply by G23M08** [reply]
welp ofrogt the reddit password so now its forgotmypasslols

**Reply by Scorpionadi** [reply]
i think that is luigi2600's remake

**Comment by nasko222** [comment]
![Screenshot_20231210-101949_YouTube.jpg](https://github.com/nasko222/the-minecraft-recreation-project/assets/52720373/d77c2bbc-cd5d-4a52-b7f1-2acaf938d7e0)

Sometimes i check TheMisterEpic's video about developer versions and i see all sorts of interesting comments but this one caught my eye. Most people there still ask when methods remakes will be public. I tend to respond to most of them but theres no good way to prove correlation to method hence he stepped out of the project now. Its nothing alarming i just found this comment funny

**Reply by nasko222** [reply]
Also I wanna add that its kinda difficult to say something like "the project is on my github" or moreso to put a link to it, youtube censors anything that considers advertising and that one SURE IS. Thanks youtube

**Comment by VVVVGoanimate** [comment]
i made some videos about some remakes on youtube, and i credited you and method, and i also mentioned you both, did you see those videos?

**Comment by VVVVGoanimate** [comment]
oh and also remember that stupid null guy XD
